advanced/editor-control-identifiers.md

```diff
@@ -192,13 +192,31 @@ tinyMCE.activeEditor.ui.registry.getAll().menuItems
-| row                    | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Table                                                                                                    | Row menu item with related controls.                                                                      |
 | tableprops             | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Table                                                                                                    | Opens the table properties dialog.                                                                        |
+| deletetable            | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Table                                                                                                    | Deletes the current table.                                                                                |
+| cell                   | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Table                                                                                                    | Cell menu item with related controls.                                                                     |
+| tablemergecells | [table]({{ site.baseurl }}/plugins/table/)   |  Table                   | Merges all currently selected cells   |
+| tablesplitcells | [table]({{ site.baseurl }}/plugins/table/)   |  Table                   | Splits merged cells   |
+| tablecellprops | [table]({{ site.baseurl }}/plugins/table/)   | Table                   | Opens the cell properties dialog  |
+| column                 | [table]({{ site.baseurl }}/plugins/table/)                                                                                         | Table                                                                                                    | Column menu item with related controls.                                                                   |
+| tableinsertcolumnbefore | [table]({{ site.baseurl }}/plugins/table/)   |  Table                   | Insert column before the currently selected column  |
+| tableinsertcolumnafter | [table]({{ site.baseurl }}/plugins/table/)   | Table                   | Insert column after the currently selected column  |
+| tablecutcolumn | [table]({{ site.baseurl }}/plugins/table/)   | Table                   | Cuts curently selected column or columns   |
```

Comment:
These 4 (cutcolumn, copycolumn, pastecolumnbefore & pastecolumnafter) shouldn't be documented yet, as we translations first, so that'll be delayed until 5.5.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1560/commits/f2d43dab7b5e437985669ad8f716958c09b18d0d